lib/exercism/use_cases/attempt.rb

```diff
@@ -15,7 +15,7 @@ def initialize(user, *stuff)
   # rubocop:enable Metrics/AbcSize
 
   def valid?
-    !!slug && Xapi.exists?(track, slug)
+    !!slug && Trackler.tracks[track]
```

Comment:
Hm. I don't think we need to change trackler, we only use this here.

Let's do this:

```
Trackler.tracks[track_id].exists? && Trackler.tracks[track_id].implementations[slug.to_s].exists?
```

It's long and kind of gross, but I until we need the null object in trackler, I'm inclined to skip adding it.